src/lib/route-segments.test.js

```diff
@@ -0,0 +1,92 @@
+/* global it, expect */
+import { getSegmentWidth,
+         getLinearValue,
+         STOPS,
+         ZERO_SPEED_STOPS
+} from './route-segments.js'
+
+describe('if speed != 0, returns segment width value using STOPS array based on zoom level', () => {
+  it('returns the first value of STOPS array if zoom level <= first zoom value and speed != 0', () => {
+    const firstZoom = STOPS[0][0]
+    const firstWidth = STOPS[0][1]
+    const result = getSegmentWidth(firstZoom - 2, 10)
+    const expected = Number(firstWidth.slice(0, -2))
```

Comment:
See comment above! Regarding replacing the `Number(x.slice(0, -2))` construction with `parseInt(x, 10)`